HW04_code.Rmd

```diff
@@ -0,0 +1,81 @@
+---
+title: "HW04_code"
+author: "Laura Hunter"
+date: "7/27/2020"
+output: git_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Cleaning the Data
+
+My hope with this assignment is to take the "2lot_results.csv" file that is in this directory and change the data in it so that it looks like the "2lot_results_clean.csv" file. Once I've done that, I'll be able to use the code I made from last week to plot Subject 8's data.
+
+```{r data cleaning}
+library(tidyverse) #load tidyverse packages
+
+unclean_2lot <- read.csv("2lot_results.csv", stringsAsFactors = FALSE) #Load raw 2lot data
+
```

Comment:
I personally would have kept the original "unclean" version of the table, just to be able to get back to it in case I wanted to.